js/layout-javascript/agenda-code.js

```diff
@@ -744,6 +748,22 @@ DynamicList.prototype.attachObservers = function() {
       $(this).parents('.agenda-item-bookmark-holder').removeClass('not-bookmarked').addClass('bookmarked');
 
       record.bookmarkButton.like();
+    })
+    .on('click', '.agenda-multiple-images-item', function() {
+      _this.imagesData.options.index = $(this).index();
+
+      var agendaImageGallery = Fliplet.Navigate.previewImages(_this.imagesData);
+
+      agendaImageGallery.listen('afterChange', function() {
+        Fliplet.Page.Context.update({
+          agendaImageGalleryId: _this.data.id,
```

Comment:
Why are we updating these page context parameters? Does this feature support opening a detail view gallery item using URL parameters? If so, I don't see any code that supports this.

Reply:
That is how it is done in the gallery component. I do not think that is it possible to open a detail view using URL, as far as I remember this is done to correctly assign images to specific component. So if we have multiple gallery components it will not show all pictures from both of those components.
![images](https://user-images.githubusercontent.com/52824207/92646011-75c1e100-f2ee-11ea-83fa-ebe4a53e61d9.PNG)

Reply:
If the page context updating isn't going to do anything for the user, then take it out. It wasn't part of the requirement to get that working.